30firstboot/firstboot-detect

```diff
@@ -12,9 +12,6 @@ fi
 
 mount -o "$opts" "$what" /sysroot
 
-# Handle x-initrd.mount without initrd-parse-etc.service
```

Comment:
This would break firstboot detection on systems with t-u < 5

Reply:
Added the code for backward compatibility again and adjusted the code accordingly to skip the new /etc bind mount.